pandas/core/arrays/base.py

```diff
@@ -1455,7 +1455,8 @@ def tolist(self) -> list:
         """
         if self.ndim > 1:
             return [x.tolist() for x in self]
-        return list(self)
+        # faster than list(self)
+        return self.to_numpy(dtype="object").tolist()
```

Comment:
Not sure if this is a good idea. We try to get rid of the object conversions in eas.

Reply:
Thx, removed the object conversion and refactored the PR a bit.

Reply:
Some context: Depending on the array we get (maybe third party), materialising the numpy array can be really expensive.